lib/changelog/changelog_file.rb

```diff
@@ -38,15 +38,21 @@ def parse
         @your_contribution_here = false
         @bad_lines = []
         File.open(filename).each_line do |line|
-          # ignore lines that aren't changes
-          next unless line[0] == '*'
+          # ignore new line
+          next if line == "\n"
```

Comment:
I've changed the line to this `next if line.strip.empty?`, seems fine to me. For you?